Guide: Import a Bearer Did (#1628)
TBD54566975/developer.tbd.website: site/docs/web5/decentralized-identifiers/how-to-import-did.mdx

---
title: Import a Did
hide_title: true
sidebar_position: 7
---

# Import a DID

In this guide, you will learn how to take your [Decentralized Identifier (DID)](/docs/web5/decentralized-identifiers/what-are-dids) with you to other apps, allowing you to keep your identity consistent everywhere across various platforms. 

## Export DID

> You will start by exporting your DID. This method converts your [Bearer DID](/docs/web5/decentralized-identifiers/how-to-create-did/#create-did) into a [Portable DID](/docs/web5/decentralized-identifiers/how-to-create-did/#portable-did), a serialized and static representation suitable for storage or transfer. 

Reviewer: we should add glossary terms for Bearer DID and Portable DID and link to those

Author: added!

> You will start by exporting your DID. This method converts your [Bearer DID](/docs/web5/decentralized-identifiers/how-to-create-did/#create-did) into a [Portable DID](/docs/web5/decentralized-identifiers/how-to-create-did/#portable-did), a serialized and static representation suitable for storage or transfer. 

Reviewer: can you specify what exactly gets returned? is it a JSON?

:::note
Your Portable DID contains private key material. Always store it securely, preferably in a trusted identity wallet or password manager.
:::

<Shnip
  snippets={[
    { snippetName: 'exportDidJs', language: 'JavaScript'},
    { snippetName: 'exportDidKt', language: 'Kotlin' },
        { snippetName: 'exportDidSwift', language: 'Swift' }
  ]}
/>

## Import DID

> Now, you can import your Portable DID to a desired location using the `import` method. 

Reviewer: ```suggestion
You can import your Portable DID to an application using the `import` method. 
```
